package.json

```
@@ -11,7 +11,7 @@
   "author": "Brickblock Engineering <dev@brickblock.io>",
   "license": "MIT",
   "scripts": {
-    "health": "ts-node scripts/healthcheck.ts",
+    "health": "ts-node -T scripts/healthcheck.ts",
```

nit: generally i'd default to using the long-form arg names because they're easier to understand

```suggestion
    "health": "ts-node --transpile-only scripts/healthcheck.ts",
```

---

ok, updated in next git push